graphics/libpano13/Portfile

```diff
@@ -1,34 +1,45 @@
+PortGroup             cmake 1.1
 
-name                libpano13
-version             2.9.19
-revision            1
-categories          graphics
-platforms           darwin
-maintainers         {gmail.com:freespace @freespace}
-license             GPL-2+
+name                  libpano13
+version               2.9.21
+revision              0
+categories            graphics
+platforms             darwin
+maintainers           {gmail.com:freespace @freespace}
+license               GPL-2+
 
-description         The cross-platform library behind Panorama Tools
+description           The cross-platform library behind Panorama Tools
 
-long_description    The panorama tools are mainly used to build panoramic images from a set of overlapping images. The usability extends beyond "just" building panoramas by far though. You can, for instance, use them to render an average of multiple images to broaden the dynamic range of the images or average out noise. You can also build object movies with them, morph between images and much more.
+long_description      The panorama tools are mainly used to build panoramic images from a set of overlapping images. The usability extends beyond "just" building panoramas by far though. You can, for instance, use them to render an average of multiple images to broaden the dynamic range of the images or average out noise. You can also build object movies with them, morph between images and much more.
 
-homepage            http://panotools.sourceforge.net/
-master_sites        sourceforge:project/panotools/libpano13/libpano13-${version}/
+homepage              http://panotools.sourceforge.net/
+master_sites          sourceforge:project/panotools/libpano13/libpano13-${version}/
 
-checksums           rmd160  88b8770815d3a8bd41cfe367ae19daffcf2ec76a \
-                    sha256  037357383978341dea8f572a5d2a0876c5ab0a83dffda431bd393357e91d95a8
+checksums             rmd160  a2e4529112f09f24a2ff6bacee5031261965f279 \
+                      sha256  79e5a1452199305e2961462720ef5941152779c127c5b96fc340d2492e633590 \
+                      size    2801535
 
-depends_lib         path:include/turbojpeg.h:libjpeg-turbo \
-                    port:tiff \
-                    port:libpng \
-                    port:zlib
+cmake.generator       Ninja
+configure.args-append -DJPEG_INCLUDE_DIR=${prefix}/include \
+                      -DJPEG_LIBRARY=${prefix}/lib/libjpeg.dylib \
+                      -DPNG_INCLUDE_DIR=${prefix}/include \
+                      -DPNG_LIBRARY=${prefix}/lib/libpng.dylib \
+                      -DSUPPORT_JAVA_PROGRAMS=OFF \
+                      -DTIFF_INCLUDE_DIR=${prefix}/include \
+                      -DZLIB_ROOT=${prefix}
+patchfiles-append     patch-endian-detection.diff
 
-configure.args      --with-jpeg=${prefix} \
-                    --with-tiff=${prefix} \
-                    --with-png=${prefix} \
-                    --with-zlib=${prefix} \
-                    --without-java
```

Comment:
This was the thing you removed that I was referring to. I don't think you need the `-D` stuff you did above, just to leave this stuff in place.

Reply:
The new version only supports building via CMake, which is why I changed this. The `-D` stuff is an attempt at a literal translation from `./configure`. I did note this in the commit message, but maybe I should have given it more prominence in the PR description.

Reply:
Ah, I really didn't understand this.